packages/gatsby/cache-dir/loader.js

```diff
@@ -219,6 +225,22 @@ export class BaseLoader {
     })
   }
 
+  loadSliceDataJson(sliceName) {
+    if (this.slicesDataDb.has(sliceName)) {
+      const jsonPayload = this.slicesDataDb.get(sliceName)
+      return Promise.resolve({ sliceName, jsonPayload })
+    }
+
+    // TODO: setup correct (no)caching headers
+    const url = `/slice-data/${sliceName}.json?v=${Date.now()}`
```

Comment:
we should get rid of `v=` and ensure caching headers for slice-data is same as for page-data